build-support/bin/bootstrap_pants_pex.sh

```diff
@@ -14,5 +14,5 @@ source "${REPO_ROOT}/build-support/pants_venv"
 # shellcheck source=build-support/bin/native/bootstrap_code.sh
 source "${REPO_ROOT}/build-support/bin/native/bootstrap_code.sh"
 
-./v2 binary src/python/pants/bin:pants_local_binary || exit 1
+./v2 --no-dynamic-ui binary src/python/pants/bin:pants_local_binary || exit 1
```

Comment:
I think it's fine to keep the UI. This isn't bootstrapping Pants itself - it's bootstrapping `pants.pex` for the sake of integration tests.

Reply:
Yea, it's actually helpful here, IMO.